src/main.cpp

```diff
@@ -254,15 +270,35 @@ main(int argc, char *argv[])
     }
 #endif
 
-    if (parser.isSet(debugOption))
-        nhlog::enable_debug_log_from_commandline = true;
-
     try {
-        nhlog::init(QStringLiteral("%1/nheko.log")
-                      .arg(QStandardPaths::writableLocation(QStandardPaths::CacheLocation))
-                      .toStdString());
+        QStringList levels;
+        levels.append(qEnvironmentVariable("NHEKO_LOG_LEVEL"));
+        if (parser.isSet(logLevel)) {
+            levels.append(parser.value(logLevel));
+        } else if (parser.isSet(debugOption)) {
+            levels.append("trace");
+        }
```

Comment:
I think I would still prefer overwrite semantics, i.e. if --log-level is set, use that, else if --debug is set, use that, else use env flag.

Reply:
Oh! When you mentioned overwriting the debug option earlier, I didn't assume you wanted to overwrite the environment variable as well. Okay, I'll make it do that.